site/docs/id/README.md

```diff
@@ -102,8 +102,8 @@ Berhasil! :tada:
 
 <div style="font-size: 0.75rem; display: flex; justify-content: center;">
 
-© 2021-2024 &middot; grammY mendukung API Bot Telegram versi 7.6 yang [dirilis](https://core.telegram.org/bots/api#july-1-2024) pada tanggal 1 Juli 2024.
-(Fitur yang disorot: Media berbayar)
+© 2021-2024 &middot; grammY mendukung API Bot Telegram versi 7.7 yang [dirilis](https://core.telegram.org/bots/api#july-7-2024) pada tanggal 7 Juli 2024.
+(Fitur yang disorot: Pesan layanan untuk pengembalian bayaran)
```

Comment:
`bayaran` literally means salary, which is incorrect. The correct and commonly used term for refund is `pengembalian dana`.

```suggestion
(Fitur yang disorot: Pesan layanan pengembalian dana)
```